modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -27,27 +78,103 @@
 
 <script>
 import AposInputMixin from 'Modules/@apostrophecms/schema/mixins/AposInputMixin.js';
+import cuid from 'cuid';
+import { klona } from 'klona';
+import draggable from 'vuedraggable';
 
 export default {
   name: 'AposInputArray',
+  components: { draggable },
   mixins: [ AposInputMixin ],
+  props: {
+    generation: {
+      type: Number,
+      required: false,
+      default() {
+        return null;
+      }
```

Comment:
```suggestion
      default: null
```

I think function is required  for arrays and objects